frontend/src/views/PortaisView.vue

```diff
@@ -1,13 +1,30 @@
 <template>
-    <div class="portais-view">
-      <h1>Portais</h1>
-      <p>Aqui você verá os portais disponíveis.</p>
+    <div>
+      <AdicionarPortal @novo-portal="adicionarPortal"/>
     </div>
   </template>
   
   <script>
+  import AdicionarPortal from '@/components/AdicionarPortal.vue';
+  
   export default {
     name: "PortaisView",
+    components: {
+      AdicionarPortal
+
+    },
+    data() {
+      return {
+        portais: []
+      }
+    },
+
+    methods: {
+      adicionarPortal(novoPortal) {
```

Comment:
Cade a chamada para o backend para salvar no banco?

Reply:
Não é a linha de baixo: this.portais.push(novoPortal) ?